ui/src/pages/Logs/Logs.tsx

```diff
@@ -175,9 +230,25 @@ const logsDisplayStyle = {
   overflow: 'hidden',
   fontFamily: 'monospace',
 };
-function Row({ containerName, containerId, time, stream, log }: DockerLog) {
+
+// let containerIconColor: string;
+
+function Row({
+  containerName,
+  containerId,
+  time,
+  stream,
+  log,
+  containerLabelColor,
+  containerIconColor,
+}: DockerLog) {
   const [open, setOpen] = useState<boolean>(false);
 
+  const labelColor: string = containerLabelColor[containerId];
```

Comment:
NIT: This type should be inferred so you don't need to declare it, I think

Instead of having this extra variable declaration, you could just do `backgroundColor: containerLabelColor[containerId]` on line 280